### ISSUE TITLE
WCAG version used not shown in country policy detailed information.

### ISSUE THREAD

**Issue by maryjom** [w3c/wai-policies-prototype#34]
The WCAG version used for the law/policy used to be in the table and not in the country details, so now needs to be added to the country details.

**Comment by AndrewArch** [comment]
Even if it was in the table, it should be in the country details to make them complete
